docs/getting_started.rst

```diff
@@ -441,41 +441,45 @@ multiple updates to a single document. Note that we don't save to the server
 after each update. We only save once to the server upon exiting the ``Document``
 context manager.
 
- .. code-block:: python
+.. warning:: Uncaught exceptions inside the ``with`` block will prevent your
```

Comment:
I've just added this warning. The other changes in this file are for correctly indenting the following code blocks.